frontend/encore/dev-server.rst

```diff
@@ -54,6 +54,7 @@ method in your ``webpack.config.js`` file:
         // ...
 
         .configureDevServerOptions(options => {
+            options.http2 = true;
```

Comment:
I saw your explanation for this, but I need more convincing. Obviously the link you added - https://webpack.js.org/configuration/dev-server/#devserverhttp2 - talks about this option, but I don't see it mentioned anywhere else.